### ISSUE TITLE
bs

### ISSUE THREAD

**Issue by hardhead254** [astroworId/stockx-sniper#2]
hhbhsvve. d rnrn

**Comment by jm3466** [comment]
i agree, lets go ahead and implement that in the next major version. what are your thoughts on this strategy? @astroworId
